### ISSUE TITLE
Fail building iOS module

### ISSUE THREAD

**Issue by tlenclos** [timanrebel/Parse#27]
I know this is a work in progress, I'm trying to add it to my project and it seems that FacebookSDK is already linked (probably with Titanium). 

I'm really new to native module development, what am I missing ?
```
[TRACE] duplicate symbol _OBJC_METACLASS_$_FBViewController in:
[TRACE]     /Users/thibz/Documents/github/titanium/app1/modules/iphone/rebel.parse/0.1.0/assets/Frameworks/FacebookSDK.framework/FacebookSDK(FBViewController.o)
[TRACE]     /Users/thibz/Library/Application Support/Titanium/modules/iphone/facebook/3.2.0/libfacebook.a(FBViewController.o)
[TRACE] ld: 304 duplicate symbols for architecture i386
```

**Comment by timanrebel** [comment]
The iOS module is not ready yet. I will I unpublish it. My apologies










—

with kind regards / met vriendelijke groet,




Timan Rebel

Rebelcorp




e: timan@rebelcorp.eu

t: +31 6 46 244 542

or hop into a chat on: https://klets.co/timanrebel


















As mentor, angel or developer I help you build great companies. From your very first MVP to scaling your business.

**Comment by tlenclos** [comment]
No apologies needed, I was trying to contribute and try to use your module instead of https://github.com/ewindso/ios-parse-titanium-module.

Do you know how to avoid this conflict with the facebookSDK ?

**Comment by tlenclos** [comment]
@timanrebel any update on this ?

**Comment by timanrebel** [comment]
No, I still have to unpublished the iPhone version :) The hard part is that both the official Facebook mode and this module want to use the Facebook sdks. have not yet found out how to fix that, unfortunately

**Comment by tlenclos** [comment]
Found this post http://iamyellow.net/post/40259552423/rename-symbols-compiler-preprocessor maybe it could help ?
